enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/lock/forseti/ForsetiClient.java

```diff
@@ -273,30 +288,46 @@ public void acquireExclusive( ResourceType resourceType, long... resourceIds ) t
                     continue;
                 }
 
-                // Grab the global lock
-                ForsetiLockManager.Lock existingLock;
-                int tries = 0;
-                long waitStartMillis = clock.millis();
-                while ( (existingLock = lockMap.putIfAbsent( resourceId, myExclusiveLock )) != null )
-                {
-                    assertValid( waitStartMillis, resourceType, resourceId );
 
-                    // If this is a shared lock:
-                    // Given a grace period of tries (to try and not starve readers), grab an update lock and wait
-                    // for it to convert to an exclusive lock.
-                    if ( tries > 50 && existingLock instanceof SharedLock )
+                Locks.WaitEvent waitEvent = null;
```

Comment:
There are two empty lines in succession here. They are causing the build to fail on a checkstyle error.